mysql_scripts/test_data/FullDBTestData.sql

```diff
@@ -146,11 +154,575 @@ SET @site_site2Id = LAST_INSERT_ID();
 INSERT INTO Site (title, address, phoneNumber, appointmentOnly, createdBy, lastModifiedBy)
 	VALUES ("No walkins site", "9876 Test St. Lincoln, NE 29384", "555-999-8888", true, @user_siteAdmin1Id, @user_siteAdmin1Id);
 SET @site_site3Id = LAST_INSERT_ID();
--- end site
 
+INSERT INTO Site (title, address, phoneNumber, appointmentOnly, createdBy, lastModifiedBy)
```

Comment:
This PR is significantly easier to test with all the data present with a bunch of data, we can remove this later. But it is nice to have while testing right now.